ts/routes/image-occlusion/tools/lib.ts

```diff
@@ -288,9 +288,9 @@ export const makeShapesRemainInCanvas = (canvas: fabric.Canvas, boundingBox: fab
         const left = obj.left!;
 
         const topBound = boundingBox.top!;
-        const bottomBound = topBound + boundingBox.height!;
+        const bottomBound = topBound + boundingBox.height! + 5;
```

Comment:
Any idea why this is needed?

Reply:
The width and height less than the value is not drawn, so using that value fixes edge problems.

Reply:
I believe the root issue is the user can tap on the screen / make a very small gesture, and end up creating a rectangle only a pixel or two in width. It then doesn't get rendered on the screen, so the user ends up with an invisible box that still counts as an occlusion. Does that sound correct @krmanik? Or am I thinking of another issue?

If my memory is correct here, maybe another solution would be refusing to create the shapes if they're below a certain size?

Reply:
This is before and after screen recording, I think above is different issues, but not sure.

**Before**
<video src="https://github.com/user-attachments/assets/9ab9afd5-b723-4046-9fcc-d537ea75f56e">

**After**
<video src="https://github.com/user-attachments/assets/a223025c-0e49-4c41-8518-2d452c293257">

Reply:
@dae That's a different issue (can't remember whether it was fixed)